petab/core.py

```diff
@@ -250,7 +266,8 @@ def concat_tables(
         if isinstance(tmp_df, str):
             tmp_df = file_parser(tmp_df)
 
-        df = df.append(tmp_df, sort=False, ignore_index=True)
+        df = df.append(tmp_df, sort=False,
```

Comment:
add comment what it does? never used a rangeindex.

Reply:
Pretty sure you used it. That's the default index, if you don't set it yourself. In case we have a proper index we want to keep it, if not, we don't want two rows labeled "0".